### PR TITLE
report the capability of vaCopy support

### PR DESCRIPTION
**Pull request by XinfengZhang**
Signed-off-by: Carl Zhang <carl.zhang@intel.com>

### CODE REVIEW

File: va/va.h [hunk 1]
```diff
@@ -4950,6 +4957,11 @@ typedef enum
      * specify vaPutSurface render area if there is no drawable on the monitor
      */
     VADisplayAttribRenderRect          = 18,
+    /*
+     * adaptor / platform attributes. read only. specify whether vaCopy support on current HW
+     * the value of each bit should equal to 1 << VA_EXEC_MODE_XXX
+     */
+    VADisplayAttibCopy                 = 19,
```
Comment (dvrogozh) [inline va/va.h:4964]: Fix: s/Attib/Attrib/

File: va/va.h [hunk 2]
```diff
@@ -4828,6 +4828,13 @@ VAStatus vaDeassociateSubpicture (
  * brightness etc. in the rendering process.  The application can query what
  * attributes are supported by the driver, and then set the appropriate attributes
  * before calling vaPutSurface()
+
+ * Display attributes also are used to query/set display adaptor related information
```
Comment (dvrogozh) [inline va/va.h:4832]: Please, reformulate and add punctuation signs appropriately.

File: va/va.h [hunk 3]
```diff
@@ -4950,6 +4957,11 @@ typedef enum
      * specify vaPutSurface render area if there is no drawable on the monitor
      */
     VADisplayAttribRenderRect          = 18,
+    /*
+     * adaptor / platform attributes. read only. specify whether vaCopy support on current HW
```
Comment (dvrogozh) [inline va/va.h:4961]: > adaptor / platform attributes.

I don't understand what you mean here.

**Review by dvrogozh — CHANGES_REQUESTED** [review]

File: va/va.h (hunk 1 above)
Comment (XinfengZhang) [reply va/va.h:4964]: yes, thanks

File: va/va.h [hunk 4]
```diff
@@ -4828,6 +4828,15 @@ VAStatus vaDeassociateSubpicture (
...
  * before calling vaPutSurface()
+ *
+ * Display attributes also are used to query/set display adaptor (vaDisplay) related information
+ * These attributes does not depend on vaConfig, only related with platform or display adaptor
+ * It could not used for vaPutSurface.
+ * App could call vaQueryDisplayAttributes/vaGetDisplayAttributes to query these attributes
+ * at anytime after vaInitialize, but only could call vaSetDisplayAttributes for these attributes
+ * after vaInitialize and before any other function call if the attributes is settable.
+ * To distinguish these two types display attributes, this display adaptor related attributres
+ * should be marked as "HW attribute" in the discription
```
Comment (emersion) [inline va/va.h:4839]: Typo: "description"

File: va/va.h [hunk 5]
```diff
@@ -4828,6 +4828,15 @@ VAStatus vaDeassociateSubpicture (
  * brightness etc. in the rendering process.  The application can query what
  * attributes are supported by the driver, and then set the appropriate attributes
  * before calling vaPutSurface()
+ *
+ * Display attributes also are used to query/set display adaptor (vaDisplay) related information
+ * These attributes does not depend on vaConfig, only related with platform or display adaptor
```
Comment (emersion) [inline va/va.h:4833]: Typo: "do not"

File: va/va.h [hunk 6]
```diff
@@ -4828,6 +4828,15 @@ VAStatus vaDeassociateSubpicture (
...
+ * It could not used for vaPutSurface.
+ * App could call vaQueryDisplayAttributes/vaGetDisplayAttributes to query these attributes
+ * at anytime after vaInitialize, but only could call vaSetDisplayAttributes for these attributes
+ * after vaInitialize and before any other function call if the attributes is settable.
+ * To distinguish these two types display attributes, this display adaptor related attributres
```
Comment (emersion) [inline va/va.h:4838]: Typo: "attributes"

**Review by dmitryermilov — APPROVED** [review]

File: va/va.h [hunk 7]
```diff
@@ -4835,6 +4835,15 @@ VAStatus vaDeassociateSubpicture (
  * brightness etc. in the rendering process.  The application can query what
  * attributes are supported by the driver, and then set the appropriate attributes
  * before calling vaPutSurface()
+ *
+ * Display attributes also are used to query/set display adaptor (vaDisplay) related information.
```
Comment (dvrogozh) [inline va/va.h:4839]: Suggest to rewrite:

```
 * Display attributes can also be used to query/set platform or display adaptor (vaDisplay)
 * related information. These attributes do not depend on vaConfig, and could not be used
 * for vaPutSurface. Application can use vaQueryDisplayAttributes/vaGetDisplayAttributes
 * at anytime after vaInitialize, but (for settable attributes) vaSetDisplayAttributes should be
 * called after vaInitialize and before any other function call.
 *
 * To distinguish these two types of display attributes, display adaptor related attributes
 * should be marked as "HW attribute" in the description.
```

**Review by dvrogozh — APPROVED** [review]

**Pull request merged**